.github/workflows/docker-push.yml

```diff
@@ -0,0 +1,68 @@
+---
+name: "Dockerfile Push"
+on:  # yamllint disable-line rule:truthy
+  push:
```

Comment:
we might want to only execute it in the master branch.
add here:
```
branches:
  -master
```

Reply:
Thanks for catching this - good point!